bin/public-url.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Used by the CI build to output the bucket prefix to use as the
+ * PUBLIC_URL environement variable.
+ *
+ * (c) 2022, Micro:bit Educational Foundation and contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+const {
+  createDeploymentDetailsWithReviewPrefixes,
+} = require("@microbit-foundation/website-deploy-aws-config");
+
+const {
+  s3Config: { bucketPrefix },
+} = createDeploymentDetailsWithReviewPrefixes();
+console.log(`/${bucketPrefix}/`);
```

Comment:
i.e. the "/update-toolkit-design/" bit in URLs like
https://review-python-editor-next.microbit.org/update-toolkit-design/